endpoints/openrtb2/video_auction.go

```diff
@@ -79,7 +81,27 @@ func (deps *endpointDeps) VideoAuctionEndpoint(w http.ResponseWriter, r *http.Re
 		CookieFlag:    pbsmetrics.CookieFlagUnknown,
 		RequestStatus: pbsmetrics.RequestStatusOK,
 	}
+
+	debugQuery := r.URL.Query().Get("debug")
+	enableDebug := debugQuery == "true"
```

Comment:
Consider being 'nice' and making this a case insensitive compare.

Reply:
Updated to be case insensitive